README.md

```diff
@@ -123,6 +123,7 @@ External dependencies:
  - libfuse-dev          >= 2.8  LGPL v2.1       https://github.com/libfuse/libfuse
  - libre2-dev           >= 2013 BSD 3-clause    https://github.com/google/re2
  - libncurses5-dev      >= 5.7  MIT             https://www.gnu.org/software/ncurses/
+ - m4                   >= 1.4  GPLv3           https://www.gnu.org/software/m4/
```

Comment:
It works, sure, and is *technically* correct from a certain perspective, but it feels a bit weird.  Still, it's probably a bit prohibitive to work reverse-engineer the actual requirement.  The one thing I'd say is that if we pin an older version in CI (I've not checked), we should probably reference whatever version that is rather than just the latest.

Reply:
That's a good point about the CI scripts. I looked into the CircleCI directory, and in the Dockerfiles, no versions are specified for any of the packages. The one exception is Fuse for Alpine.

https://github.com/sifive/wake/tree/master/.circleci/dockerfiles

So I'm wondering if there's much utility in listing out versions, unless these have been previously determined in a specific way. I could always go the non-invasive route and just not specify a version for m4.

Reply:
I'd assume there was originally some basis for the others, so I'd definitely not remove anything, but without actually testing the lower versions in CI there's nothing guaranteeing the others are still valid so we probably don't have to be too worried about being specific until/unless we do add some bounds checking.  Besides, m4 >= 1.4 seems to take us earlier than 2006, so as long as they're using semantic versioning it seems to be quite good enough.